modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form', 1),
```

Comment:
I think to use the node load auto load you probably need a different page callback -- see that article i linked.  The idea is that you won't have to do a node load as a result it should already be there and loaded.

https://www.drupal.org/node/224170

Now take a further look there -- they actually give you some bad info, its not a trick, but you do have to read through -- we need a page callback so we can make use of the autoloader. We want the autoloader -- because in a way it sets you up for its own style of validation.